packages/remix-react/links.ts

```diff
@@ -173,23 +173,11 @@ interface HtmlLinkPreloadImage extends HtmlLinkProps {
 export type HtmlLinkDescriptor =
   // Must have an href *unless* it's a `<link rel="preload" as="image">` with an
   // `imageSrcSet` and `imageSizes` props
-  (
     | (HtmlLinkProps & Pick<Required<HtmlLinkProps>, "href">)
     | (HtmlLinkPreloadImage &
         Pick<Required<HtmlLinkPreloadImage>, "imageSizes">)
     | (HtmlLinkPreloadImage &
         Pick<Required<HtmlLinkPreloadImage>, "href"> & { imageSizes?: never })
```

Comment:
Sorry, made a mistake with the formatting fix:

```suggestion
  | (HtmlLinkProps & Pick<Required<HtmlLinkProps>, "href">)
  | (HtmlLinkPreloadImage & Pick<Required<HtmlLinkPreloadImage>, "imageSizes">)
  | (HtmlLinkPreloadImage &
      Pick<Required<HtmlLinkPreloadImage>, "href"> & { imageSizes?: never });
```